core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -523,13 +517,11 @@ private[ui] class JobPagedTable(
     store,
     data,
     basePath,
-    currentTime,
```

Comment:
If we can remove `currentTime` here, can we also remove `currentTime` from `JobPagedTable`?

Reply:
yeah. It can be removed. Thanks.